bin/ember-template-lint.js

```diff
@@ -254,6 +254,10 @@ function getTodoConfigFromCommandLineOptions(options) {
   return todoConfig;
 }
 
+function isOverridingConfig(options) {
+  return options.config || options.rule || options.noInlineConfig || options.noConfigPath;
```

Comment:
I think we also need to look at `options.configPath`

Reply:
Done!